portfolioProject_2018-10-02.sql

```diff
@@ -0,0 +1,54 @@
+# ************************************************************
```

Comment:
why so many databases? just wastes space in git repo, have one with an un-dated name and override it each time